lib/radiator/auth/register.ex

```diff
@@ -6,11 +6,28 @@ defmodule Radiator.Auth.Register do
   alias Radiator.Repo
   alias Radiator.Auth.User
 
-  def get_user(id), do: Repo.get!(User, id)
+  def get_user(id),
+    do:
+      Repo.get!(User, id)
+      |> preload()
 
-  def get_user_by_email(email), do: email |> User.by_email_query() |> Repo.one()
+  def get_user_by_email(email),
+    do:
+      email
+      |> User.by_email_query()
+      |> Repo.one()
+      |> preload()
 
-  def get_user_by_name(name), do: name |> User.by_name_query() |> Repo.one()
+  def get_user_by_name(name),
+    do:
+      name
+      |> User.by_name_query()
+      |> Repo.one()
+      |> preload()
+
+  def preload(user) do
+    Repo.preload(user, :person)
```

Comment:
Right, but you need to be able to get the person for a user -- what's the alternative?

Reply:
I would suggest that we don't need to create a person when registering.
A user should in my opinion not rely on a person.

A person can optionally reference a user so that the system knows when a person is the one currently logged in.

But i would let a user has all properties that are necessary to register.
I think that would be `email`, `username` (not sure if that is necessary), `password`